src/v2/view-builder/views/consent/EmailMagicLinkOTPTerminalView.js

```diff
@@ -0,0 +1,59 @@
+import { loc, View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+
+const getTerminalOtpEmailMagicLinkContext = (appState) => {
+  const appName = loc('idx.return.link.otponly.accessing.app', 'login', [appState.get('app').label]);
+  const client = appState.get('client');
+  const browserOnOsString = loc('idx.return.link.otponly.browser.on.os', 'login', [client.browser, client.os]);
+  // need city and 1 of (state, country) to display location
+  const clientLocationExists = client.location &&
+   client.location.city && 
+   (client.location.state || client.location.country);
+  const geolocation = clientLocationExists ?
+    loc('idx.return.link.otponly.city.state.country', 
+      'login',
+      [client.location.city, client.location.state || client.location.country]) :
+    null;
+  const otp = appState.get('currentAuthenticator').contextualData.otp;
+
+  return {
+    appName,
+    browserOnOsString,
+    geolocation,
+    otp,
+  };
+};
+
+const BaseEmailMagicLinkOTPTerminalView = View.extend({
+  getTemplateData() {
+    return getTerminalOtpEmailMagicLinkContext(this.options.appState);
+  },
+});
+
+const OTPInformationTerminalView = BaseEmailMagicLinkOTPTerminalView.extend({
+  template: hbs`
+  <h1 class='otp-value'>{{otp}}</h1>
+  <div class="enduser-email-consent--info no-translate">
+    <div>{{i18n code="idx.return.link.otponly.request" bundle="login"}}</div>
+  </div>
+  <div class="enduser-email-consent--info no-translate">
+    <i class="enduser-email-consent--icon icon--desktop"></i>
+    <div>{{browserOnOsString}}</div>
+  </div>
+  <div class="enduser-email-consent--info no-translate">
+    <i class="enduser-email-consent--icon icon--app"></i>
```

Comment:
Can you remind me if these icons are presented to screen readers? I don't believe they are vocalized, which is likely the experience we want for this page.

Reply:
That's a very good point, is there an example of how to get these icons vocalized?

Reply:
I'd assume we need to add the `aria-hidden="true"` to the `<i>` component to make sure it is hidden from screen readers. To validate this, we can load the page and turn on [VoiceOver](https://support.apple.com/guide/voiceover/turn-voiceover-on-or-off-vo2682/10/mac/10.15), navigating through the rendered form.

Maybe loop in your designer for feedback/requirements on a11y?

Reply:
Just added it and tried it out, the screen reader skipped over the icons! (First time trying it, that was pretty cool). I'll run it by our designer before merging! Thanks for all the help @jmelberg-okta